HISTORY.md

```diff
@@ -1,5 +1,5 @@
 ## History
 
-### 1.0.0
+### 0.1.0
```

Comment:
no it should be 1.0.0 but no need to change it in this commit since it's not a release